packages/mcp/mcp-client-chat/src/mcp-client-chat.ts

```
@@ -1,50 +1,55 @@
+  ToolResults,
 } from './type.js';
 
+const DEFAULT_AGENT_STRATEGY = AgentStrategy.FUNCTION_CALLING;
 export class McpClientChat {
   protected options: MCPClientOptions;
   protected iterationSteps;
   protected clientsMap: Map<string, Client> = new Map();
   protected toolClientMap: Map<string, Client> = new Map();
-  protected messages: Message[];
+  protected messages: Message[] = [];
   protected transformStream: TransformStream = new TransformStream();
```

ESLint

[error] The 'TransformStream' is still an experimental feature The configured version range is '^20.19.0 || >=22.12.0'.

(n/no-unsupported-features/node-builtins)